.github/workflows/tests.yml

```
@@ -87,7 +87,7 @@ jobs:
           # - {device: "default.qubit", analytic: false, shots: 1000}
           # - {device: "default.qubit.tf", analytic: true, shots: 1000}
           - {device: "default.qubit.autograd", analytic: true, shots: 1000}
-          - {device: "default.mixed", analytic: true, shots: 1000}
+          - {device: "default.mixed", analytic: true, shots: 1000, interfaces: ['jax']}

```

Note: wrong suggestion from my side
```suggestion
          - {device: "default.mixed", analytic: true, shots: 1000}

```